integration/tests/cypress_e2e.sh

```diff
@@ -47,6 +49,6 @@ do_test_deploy() {
     npm install # get dependencies defined in e2e/package.json
     if ! npm run cypress:run; then
         buildkite-agent artifact upload "cypress/videos/*;cypress/videos/**/*;cypress/screenshots/*;cypress/screenshots/**/*"
-        return 1
+        exit 1
```

Comment:
The situation here is different, though -- **this defines a function**, and `return 1` is thus the right thing to do, I'd say. See:

```
$ cat script_three.sh
#!/bin/bash

foo() {
  return 1
}

foo
$ ./script_three.sh
$ echo $?
1
$
```